src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationOptions.cs

```diff
@@ -28,5 +29,12 @@ public GoogleAuthenticationOptions()
         /// access_type. Set to 'offline' to request a refresh token.
         /// </summary>
         public string AccessType { get; set; }
+
+        /// <summary>
+        /// Defines whether access and refresh tokens should be stored in the
+        /// <see cref="ClaimsPrincipal"/> instance after a successful authentication.
+        /// This option is turned off by default to reduce the size of the final authentication cookie.
+        /// </summary>
+        public override bool SaveTokensAsClaims { get; set; } = false;
```

Comment:
So what I don't like about this vs the old pattern is its pretty hard to tell what's changing, before in the ctor, you can easily see which defaults were affected just by looking at the ctor.  Now you have to wade through all of the various inline'd autoproperties, its just not as obvious.  I'm not sure its worth it just for doc comments.  I am sort of leaning towards going back to simple auto properties and derived classes can just change the defaults in the ctor, you can continue to put any doc comments about the defaults changing in the ctor still.  Its also just less code...  @Tratcher thoughts?